zuul.d/projects.yaml

```diff
@@ -13,3 +13,4 @@
         - edpm-ansible-molecule-edpm_frr
         - edpm-ansible-molecule-edpm_iscsid
         - edpm-ansible-molecule-edpm_ovn_bgp_agent
+        - edpm-ansible-molecule-neutron-dhcp
```

Comment:
should the name be in line with the rest of jobs?

Reply:
Good catch. Fixed.